docs/admin/out-of-resource.md

```diff
@@ -35,20 +39,68 @@ summary API.
 | `imagefs.available` | `imagefs.available` := `node.stats.runtime.imagefs.available` |
 | `imagefs.inodesFree` | `imagefs.inodesFree` := `node.stats.runtime.imagefs.inodesFree` |
 
-Each of the above signals supports either a literal or percentage based value.  The percentage based value
-is calculated relative to the total capacity associated with each signal.
+The value for `memory.available` is derived from the cgroupfs instead of tools
+like `free -m`.  This is important because `free -m` does not work in a
+container, and if users use the [node
+allocatable](/docs/admin/node-allocatable.md) feature, out of resource decisions
+are made local to the end user pod part of the cgroup hierarchy as well as the
+root node.  The following script simulates the same set of steps that the
+`kubelet` performs to calculate `memory.available`.  The `kubelet` excludes
+inactive_file (i.e. # of bytes of file-backed memory on inactive LRU list) from
+its calculation as it assumes that memory is reclaimable under pressure.
+
+```sh
```

Comment:
Can we move this script to the end of the doc instead or better make it a separate file and link to it?